lib/stanford-mods/date_parsing.rb

```diff
@@ -36,6 +37,7 @@ def self.year_str_valid?(year_str)
 
       def initialize(date_str)
         @orig_date_str = date_str
+        @orig_date_str.freeze
```

Comment:
Hmm, good point.  I *think* that in this context it doesn't particularly matter, but what you've done is probably more explicit in this case.  This is probably is more clear that we're not attempting to freeze the parameter that was sent in.

:+1: